webhook_to_fedora_messaging/main.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+The values for any configuration variable that was not mentioned of in the
+custom configuration file will be inherently taken from the default values
+"""
+
+
+from flask import Flask
+
+from webhook_to_fedora_messaging.config.config import return_flaskapp_config_data
+from webhook_to_fedora_messaging.config.defaults import LOGGER_CONFIG
```

Comment:
FYI, you can also do `from .config.defaults import LOGGER_CONFIG`, if you want to save the typing ;-)